src/components/MultiChoiceResult.vue

```diff
@@ -18,5 +18,10 @@ import { Component, Vue, Prop } from "vue-property-decorator";
 @Component
 export default class MultiChoiceResult extends Vue {
   @Prop() data: any;
+  getValue(str: string): string {
```

Comment:
Is it possible that this change introduced a change in the way the visibleIf property handles the validation of question 7?

See [line 841 of JSON file](https://github.com/canada-ca/aia-eia-js/blob/7927e05b3d5aba618cb1f12f4ecac26178c919c5/src/survey-enfr.json#L841)

Reply:
Nope, issue is with multichoice answers: with current **visibleIf** property setup:

`"visibleIf": "{question7} = [\"item6\"]"`

If you select more than just item 6, the question 8 won't be displayed.

So the fix works

Reply:
The problem was with the JSON itself. Replace `=` by `contains` and works as intended.